cli/Cargo.toml

```diff
@@ -32,7 +32,7 @@ maplit = "1.0.2"
 minisign-verify = "0.2.1"
 nom = "7.1.1"
 open = "2.1.2"
-phylum_types = { git = "https://github.com/phylum-dev/phylum-types", branch = "development" }
+phylum_types = { path = "/home/chris/programming/rust/phylum-types" }
```

Comment:
is this right?

Reply:
Yes, the PR is currently up at phylum-types: https://github.com/phylum-dev/phylum-types/pull/33

Reply:
just the path to chris's home seems a bit dodgy...

Reply:
Yeah well that's my user's home directory. Unless your user is also named `chris` and you're following my directory structure, that path would probably have to be changed for testing.

Could also link against the branch of course, but there's no point in making this PR work since that would only risk accidentally merging it.

Reply:
i just realized this is NOT an actual PR... oh god. sorry!